docs/en/operations/settings/settings_users.md

```diff
@@ -0,0 +1,125 @@
+# User settings
+
+The `users` section of the `user.xml` configuration file aggregates settings for users.
```

Comment:
"aggregates" is not really a fitting verb as common pattern is to put each user into separate file, each having <users> section